services/kommander/0.6.0/dynamic-helmreleases/mtls-proxy/defaults/cm.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mtls-proxy-0.1.8-d2iq-defaults
```

Comment:
Why there is `0.1.8` in the name?

Reply:
that's from the mtls-proxy chart version

https://github.com/mesosphere/kommander-applications/blob/9cebeb79ac009fe39b0382b66cc60a23f42039d2/services/kommander/0.6.0/dynamic-helmreleases/mtls-proxy/mtls-proxy.yaml#L19